engine/release-notes/index.md

```diff
@@ -22,6 +22,62 @@ for Docker Engine.
 
 # Version 20.10
 
+## 20.10.8
+2021-08-03
+
+### Deprecation
+
+- Deprecate support for encrypted TLS private keys.
+  Legacy PEM encryption as specified in RFC 1423 is insecure by design. Since
+  it does not authenticate the ciphertext, it is vulnerable to padding oracle
```

Comment:
```suggestion
  it does not authenticate the ciphertext, it is vulnerable to padding Oracle
```

Reply:
I think `oracle` here is "oracle" (not the company/product). I recall I copied this text from Golang's description of the issue (Golang deprecated this functionality).

Reply:
Aah, makes sense. Thanks for clarifying.